documentation/plain_rules.md

```diff
@@ -104,16 +104,33 @@ Sends an email to the recipient set in the action parameters, with the body mail
 ```
 
 ### update attribute action
-Updates an specified attribute of the entity which caused the rule to be fired.
-This action does not use any value from the generated event  (except the id from the originating entity, but this is done transparently). The `parameters` field includes a field `name` for the name of the attribute to be changed and a `value` field with the value to be put in the attribute.
+Updates an specified attribute of the entity/event which caused the rule to be fired or another one.
+The `parameters` field must include a field `name` for the name of the attribute to be changed and a `value` field with
+the value to be put in that attribute.
 
+Optionally, an `id` attribute can be set in `parameters`. The value can be a literal value or a template in which an expression like `${X}`
+ will be substituted by the value of the attribute `X` from the entity firing the rule. So, if we want the entity with "id" `sensor1`
+to update the entity with "id" `sensor1_friend` we should set the `id` field in `parameters` with the value `"${id}_friend"`
+
+The parameters for this action:
+* **name**: *mandatory*, attribute name to set
+* **value**: *mandatory*, attribute value to set
+* id: optional, the source entitity's id by default
+* type: optional, the source entitity's type by default
+* isPattern: optional, `false` by default
```

Comment:
Currently, Orion doesn't support updateContext with isPattern=true. However, we can keep the parameters in CEP, as it could be implemented in the future in Orion.

NTC